Sources/Fuzzilli/Core/CodeGenerators.swift

```diff
@@ -393,6 +393,32 @@ public let CodeGenerators: [CodeGenerator] = [
         b.reassign(target, to: val)
     },
 
+    CodeGenerator("DestructArrayGenerator", input: .iterable) { b, arr in
+        // Fuzzilli generated arrays can have a length ranging from 0 to 10 elements,
+        // We want to ensure that 1) when destructing arrays we are within this length range
```

Comment:
Nit: maybe "when destructing arrays we are usually within", because we might still generate arrays with only 1 or 2 elements